app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -2310,6 +2310,53 @@ class DatabaseMigrationAndroidTest {
     verifyNewStatus(uuid = uuid2, newStatus = "allowed")
     verifyNewStatus(uuid = uuid3, newStatus = "denied")
   }
+
+  @Test
+  fun migrate_userstatus_from_37_to_38() {
+    val db_v37 = helper.createDatabase(version = 37)
+
+    fun insertAppointment(uuid: String, oldStatus: String) {
+      db_v37.execSQL("""
+        INSERT OR REPLACE INTO `Appointment`
+          (`uuid`,`patientUuid`,`facilityUuid`,`scheduledDate`,`status`,`appointmentType`,`syncStatus`,`createdAt`,`updatedAt`)
```

Comment:
We can also simplify this and add all the values.
We wont need to pass column names then.

Reply:
I'd like to keep these because then later when the model changes it will be easier to know what value corresponds to what key in an older migration.

Reply:
So for that we actually name the values same as the column name.
Although its ok for now.